html/images/logos/hatteras.svg

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="200"
```

Comment:
html/images/logos/hatteras.svg: SVG files must not contain length or width attributes . This is verified automatically and makes the test fail. Could you please remove those from all the SVG files ? 
Thanx

Reply:
fixed